src/components/invitations/InvitationsAPI.js

```diff
@@ -0,0 +1,25 @@
+import { alertError } from "../../store/actions/alertActions";
+import axios from "axios";
+import store from "../../store";
+
+const API_HOST = process.env.REACT_APP_API_PROJECTS_URL;
+const API_NAME = "projects";
+const API_VERSION = "v1";
+
+export async function getInvitationsByProject(id) {
+  try {
+    const { data } = await axios.get(
+      `${API_HOST}/api/${API_VERSION}/${API_NAME}/${id}/invitations`
+    );
+    return data;
+  } catch (error) {
```

Comment:
Improve your error handling here or in the component. I always get the following error when entering a project details. Note that if the project doesn't have any invitations... it isn't an error. Check if the problem is here or from the API
![image](https://user-images.githubusercontent.com/85913826/129971792-5040d02e-7ca8-4112-a97c-f76b8385bc60.png)

Reply:
![image](https://user-images.githubusercontent.com/72277383/129985541-9323e8b4-d245-4a98-8c3e-8277c16f7be4.png)
In projects details I got 200 as response of the request and doesnt appear the alert error.